_layouts/front.html

```diff
@@ -54,26 +54,26 @@
 
 			<p>
 				If you are a developer and want to try out the tool chain then start
-				with the <a href="book/200-quick-start.html">base tutorial</a>.
+				with the <a href="book/220-tutorial-base.html">base tutorial</a>.
 				This is a zero-knowledge required tutorial that guides you through
 				all the principles of developing with enRoute. This tutorial will
 				work you through all the basics.
 			<p>
 				If you are an architect and want to know more about what enRoute is
 				and does then you start reading the <a href="book/210-overview.html">overview</a>
-				of the enRoute project
+				of the enRoute project.
 			<p>
-				If you're interested in the enRoute Profiles, you can find more to
-				read in the <a href="book/400-datasheets.html"></a>
+				<!-- If you're interested in the enRoute Profiles, you can find more to
+				read in the <a href="book/400-datasheets.html"></a> -->
```

Comment:
I don't think in this instance it is a matter of a missing page with this link serving as a placeholder. From what I can tell, the "datasheets" idea has evolved into the "services" thinking. To my mind, it is a question of @pkriens intent and terminology which I don't think I or any other potential contributor can provide. That's what I see in this situation.